cognite/powerops/custom_modules/power_model_v1/data_models/views/function_inputs/1-interface.PartialBidMatrixCalculationInput.view.yaml

```diff
@@ -0,0 +1,43 @@
+space: "{{powerops_models}}"
+externalId: PartialBidMatrixCalculationInput
+name: PartialBidMatrixCalculation
+description: The data for a bid calculation for water value based method
+filter:
+  and:
+    - hasData:
+        - type: container
+          space: "{{powerops_models}}"
+          externalId: FunctionData
+    - equals:
+        property:
+          - node
+          - space
+        value: "{{powerops_instance_space}}"
+implements:
+  - space: "{{powerops_models}}"
+    externalId: FunctionInput
+    version: "{{version}}"
+    type: view
+version: "{{version}}"
+properties:
+  bidDate:
+    container:
+      space: "{{powerops_models}}"
+      externalId: FunctionData
+      type: container
+    containerPropertyIdentifier: date1
+    name: bidDate
+    description: The bid date
+  bidConfiguration:
```

Comment:
I know we discussed the need for this (to be able to connect it to the bidDocument, right?), but still seems a bit strange to both provide the entire bidconfig AND the partial config for the specific SHOP/Water partialInput. I guess it depends if we create the bid document with the connection to the bidconfig in the calculate_total_bid_matrix function, or in the beginning of the workflow in the taskdispatcher. Then it would suffice to only give in partialConfig, priceArea and marketconfig here? 

Just wondering if we can solve this a bit more neatly maybe